crates/tracing/src/traces.rs

```diff
@@ -166,9 +192,23 @@ pub fn configure_tracing(
     let stderr = std::io::stderr();
     let ansi = stderr.is_terminal();
     let (stderr, stderr_guard) = tracing_appender::non_blocking(stderr);
+
+    let mut error_counter = ErrorCounter::default();
```

Comment:
Where does the error counter get incremented at? We have something that reads the value in the filter, but I don't see anywhere that is keeping track of the errors and incrementing